jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <j-tray-plugin
-    :description="docs_description || 'Tools for selecting and interacting with subsets.'"
```

Comment:
Looks like this accidentally got picked up. You can set the `docs_description` in the plugin itself so we can get rid of the text here.